bin/check-annotations

```diff
@@ -4,45 +4,108 @@ Check annotations for potential errors / missing data
 """
 
 ## Not part of this script, but you can check if a strain has duplica
-# 
 
-import pandas as pd
 import sys
+import pandas as pd
+
+ANNOTATIONS_FILE = "./source-data/annotations.tsv"
```

Comment:
Before merging this, we'll want to stop relying on a file path relative to this script but instead relative to the base of this repo. e.g like what we've done in [transform](https://github.com/nextstrain/ncov-ingest/blob/6d4b6de889fbc54a3cba220cf4b813587242fc60/bin/transform#L251).

Reply:
Done! I've also implemented this with argparse (like in transform), so it's easier to specify a different file name (helped with testing).

Reply:
Awesome. Thanks for doing this.